routers/api/v2/tokens.go

```diff
@@ -64,3 +67,55 @@ func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
 		Token: token,
 	})
 }
+
+// GET: /api/v2/tokens/resources/authorized?from={authorisedUris}
+// Request:	 authorisedUris string
+// Response: authorisedUris []v2.UniformResourceIdentifier
+// Headers:  Authorization <- token
+func (r *apiV2Router) GetAuthorizedResources(ctx *gin.Context) {
+	fromUris := ctx.Query("from")
+	if len(fromUris) == 0 {
+		r.logger.Debug("could not parse get authorized resources request")
+		models.SendAPIError(ctx, http.StatusBadRequest, "failed to parse request")
+		return
+	}
+
+	fromUris, err := url.QueryUnescape(fromUris)
```

Comment:
Don't we unescape the URIs during unmarshaling?

Reply:
Not at the moment, we should probably change it so we do the unescape in the unmarshalling

Reply:
The unmarshall function in `UniformResourceIdentifier` unmarshalls a single URI. We need to unescape the whole query parameter here since we don't know if we've been provided one, or multiple URIs in the request

Reply:
Right, so we're unmarshalling symbols like commas here?

Reply:
Exactly, and any symbols that are included in the URI's arguments

Reply:
Ok, makes sense